### PR TITLE
feat(docs): new language lead handbook

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

After having discussed it on chat, and noticing certain questions coming up often, it was time to start this documentation.
To be updated as new questions come up.

### CODE REVIEW

File: docs/language-lead-handbook.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+The Crowdin Glossary is kept in the [crowdin-glossaries](https://github.com/freeCodeCamp/crowdin-glossaries) repository.
+
+In the `glossaries` folder there is various `*.csv` (comma separated values) files, one for each of the crowdin projects that have a glossary that can be updated from this workflow.
+
+The `client.csv` file is for the Learn User Interface project, the `curriculum.csv` file is for the Coding Curriculum project, the `docs.csv` file is for the Contributing Documentation project.
+
+To update the Crowdin Glossaries you need to clone this repo locally. Open the `.csv` file with an appropriate program, for example Microsoft Excel.
+
+In the `.csv` file you will find that the English language occupies the first three columns, `Term:English` is the column for the English term, `Description:English` us the column for the English description, and `Part:English` is for the grammar role, or part of the sentence, of the term.
```
Comment (Dario-DC) [inline docs/language-lead-handbook.md:30]:
```suggestion
In the `.csv` file you will find that the English language occupies the first three columns, `Term:English` is the column for the English term, `Description:English` is the column for the English description, and `Part:English` is for the grammar role, or part of the sentence, of the term.
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

**Review by Sembauke — DISMISSED** [review]

**Review by DanielRosa74 — DISMISSED** [review]
LGTM

File: docs/language-lead-handbook.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+In the `.csv` file you will find that the English language occupies the first three columns, `Term:English` is the column for the English term, `Description:English` is the column for the English description, and `Part:English` is for the part of speech (e.g., noun, verb etc.) of the term.
+
+Then, each target language has two columns. If you translate to Dothraki, you will be interested in the columns `Term:Dothraki` and `Description:Dothraki`. The column `Term:Dothraki` is for the translation of the term in Dothraki, and the column `Description:Dothraki` is for a description of the term in Dothraki.
+
+> [!TIP]
+> In programs like Microsoft Excel you can hide the columns of the other languages to free screen real-estate and see the English columns and the target language columns near each other.
+
+After you have made the changes and saved the file, you will need to make a PR with the proposed changes. After the PR is accepted, the script to update the Crowdin Glossary. Your glossary changes will not have immediate effects, but they will come.
```
Comment (nhcarrigan) [inline docs/language-lead-handbook.md:49]:
```suggestion
After you have made the changes and saved the file, you will need to make a PR with the proposed changes. After the PR is accepted, you will need to run the GitHub Action workflow to update the Crowdin Glossary. Your glossary changes will not have immediate effects, but they will come.
```

File: docs/language-lead-handbook.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+The `client.csv` file is for the Learn User Interface project, the `curriculum.csv` file is for the Coding Curriculum project, the `docs.csv` file is for the Contributing Documentation project.
+
+To update the Crowdin Glossaries you need to clone this repo locally. Open the `.csv` file with an appropriate program, for example Microsoft Excel.
+
+In the `.csv` file you will find that the English language occupies the first three columns, `Term:English` is the column for the English term, `Description:English` is the column for the English description, and `Part:English` is for the part of speech (e.g., noun, verb etc.) of the term.
+
+Then, each target language has two columns. If you translate to Dothraki, you will be interested in the columns `Term:Dothraki` and `Description:Dothraki`. The column `Term:Dothraki` is for the translation of the term in Dothraki, and the column `Description:Dothraki` is for a description of the term in Dothraki.
+
+> [!TIP]
+> In programs like Microsoft Excel you can hide the columns of the other languages to free screen real-estate and see the English columns and the target language columns near each other.
```
Comment (nhcarrigan) [inline docs/language-lead-handbook.md:47]:
```suggestion
> In programs like Microsoft Excel you can hide the columns of the other languages to free up screen real-estate and see the English columns and the target language columns near each other.
```

File: docs/language-lead-handbook.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  const fccOriginalPost = 'link';
+</script>
+```
+
+With `link` being the link of the original article.
+
+## How to update Crowdin Glossary
+
+> [!TIP]
+> An updated glossary helps in having an homogeneus translation of technical terms.
```
Comment (nhcarrigan) [inline docs/language-lead-handbook.md:32]:
```suggestion
> An updated glossary helps in having an homogenous translation of technical terms.
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: docs/language-lead-handbook.md (hunk 4 above)
Comment (ieahleen) [reply docs/language-lead-handbook.md:32]:
```suggestion
> An updated glossary helps in having an homogeneous translation of technical terms.
```

**Review by naomi-lgbt — DISMISSED** [review]

**Review by sidemt — COMMENTED** [review]
I'm sorry my review was in pending status. I suggest adding a few things.

**Review by naomi-lgbt — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Sembauke — APPROVED** [review]

**Pull request merged**